DigitalLearningSolutions.Web.IntegrationTests/AuthenticationTests.cs

```diff
@@ -1,36 +1,53 @@
-namespace DigitalLearningSolutions.Web.IntegrationTests
-{
-    using System.Threading.Tasks;
-    using Microsoft.AspNetCore.Mvc.Testing;
-    using Xunit;
-
-    public class AuthenticationTests: IClassFixture<CustomWebApplicationFactory<Startup>>
-    {
-        private readonly CustomWebApplicationFactory<Startup> _factory;
-
-        public AuthenticationTests(CustomWebApplicationFactory<Startup> factory)
-        {
-            _factory = factory;
-        }
-
-        [Theory]
-        [InlineData("/Home")]
-        [InlineData("/Login")]
-        [InlineData("/ForgotPassword")]
-        [InlineData("/LearningSolutions/AccessibilityHelp")]
-        [InlineData("/LearningSolutions/Terms")]
-        public async Task EndpointIsUnauthenticated(string url)
-        {
-            // Arrange
-            var client = _factory.CreateClient(new WebApplicationFactoryClientOptions {
-                AllowAutoRedirect = false
-            });
-
-            // Act
-            var response = await client.GetAsync(url);
-
-            // Assert
-            response.EnsureSuccessStatusCode();
-        }
-    }
-}
+namespace DigitalLearningSolutions.Web.IntegrationTests
+{
+    using System.Threading.Tasks;
+    using FluentAssertions;
+    using Microsoft.AspNetCore.Mvc.Testing;
+    using Xunit;
+
+    public class AuthenticationTests: IClassFixture<CustomWebApplicationFactory<Startup>>
+    {
+        private readonly CustomWebApplicationFactory<Startup> _factory;
+
+        public AuthenticationTests(CustomWebApplicationFactory<Startup> factory)
+        {
+            _factory = factory;
+        }
+
+        [Theory]
+        [InlineData("/Home/Section/Welcome")]
+        [InlineData("/Home/Section/Products")]
+        [InlineData("/Home/Section/LearningContent")]
+        [InlineData("/Login")]
+        [InlineData("/ForgotPassword")]
+        [InlineData("/LearningSolutions/AccessibilityHelp")]
+        [InlineData("/LearningSolutions/Terms")]
+        public async Task EndpointIsUnauthenticated(string url)
+        {
+            // Arrange
+            var client = _factory.CreateClient(new WebApplicationFactoryClientOptions {
+                AllowAutoRedirect = false
+            });
+
+            // Act
+            var response = await client.GetAsync(url);
+
+            // Assert
+            response.EnsureSuccessStatusCode();
+        }
+
+        [Fact]
+        public async Task BaseUrlRedirectsToWelcomePage()

```

Comment:
I don't think this belongs in AuthenticationTests, but I don't know whether to change the class name or move this test into a separate class. Thoughts?

Reply:
For now, I think I'd keep it here and change the class name as it is all testing the Microsoft Identity stuff. If this class starts getting too big, we can separate it out. :)